src/components/tooltip/README.md

```diff
@@ -71,10 +71,26 @@ const Body = styled.div`
 </Tooltip>;
 ```
 
+### Only displaying the tooltip under certain conditions
+
+There may be cases when you only want to enable the display of a tooltip under a certain condition. In these cases, you may want to set the `isEnabled` prop to false.
+
+In the following example, the tooltip text only appears on hover when the button is disabled.
```

Comment:
Reading this I like this more than `isVisible` cause the latter may spark the understanding that it is forced to be visible even without hovering. Maybe `isSurpressed` is another name for the prop?

Reply:
@emmenko do you have any opinions on the prop naming?

* isDisabled
* isEnabled
* isSurpressed

Reply:
I liked isSurpressed but am leaning towards isDisabled for API consistency.

Reply:
Hmm, if you end up doing `isEnabled={!props.isDisabled}` maybe `isDisabled` makes more sense?
I'm also thinking about something like `off`, which maybe better indicates that the event listeners are turned off.

Reply:
I renamed it to `isDisabled`. The other option could be isOff 🤔.

Reply:
Or just `off` 🤔

Reply:
all of our other props start with `is` though

Reply:
I'm okay with `off` if @tdeekens is as well. I just want to :shipit:

Reply:
`off` is fine. I kind of don't like to break with patterns unless there is a good reason. So my choice would have been `isOff` even if that is more to write.